website/docs/reference/data-test-configs.md

```diff
@@ -275,3 +275,23 @@ tests:
 ```
 
 </File>
+
+#### Specify custom configurations for generic data tests
+
+_Currently available in dbt Cloud only. The `custom_config` key will become available in dbt Core later this year._
```

Comment:
```suggestion
_Currently available in dbt Cloud only. Specifying custom configurations for data tests will become available in dbt Core later this year._
```

Reply:
@nghi-ly Could we also add a Release Note for this? It should include the same callout re: dbt Cloud / Core availability